magma/bits.py

```diff
@@ -505,23 +505,28 @@ def __lt__(self, other):
 
     @classmethod
     def get_family(cls):
-        return m.get_family()
+        return get_family()
+
+    @classmethod
+    def unflatten(cls, value):
+        # TODO(rsetaluri): Should we type check here?
+        return Bits[len(value)](value)
```

Comment:
I think it'd be fine to assume that the constructor type checks, no need to do it in two places.

Reply:
The issue is if we do `Bits[8].unflatten([x, y, z])`, i.e. size-mismatch. Right now it will indiscriminately return `Bits[3]()`. This doesn't seem to bad so just added a logger warning and got rid of TODO.